### ISSUE TITLE
DAC retest - fix issues with downloable resources

### ISSUE THREAD

**Issue by boxadesign** [ONSdigital/design-system#2474]
The DAC retest failed the downloadable resources pattern on the following:

- Page 17 - Dialog - Screen reader mobile users that activate the ‘Show filters’ button are not trapped to the 
filter dialog that appears. This means that users are able to navigate to the content that is 
behind the dialog. Visual screen reader users may become disorientated without trapping 
the users focus to the dialog.

- Page 19 - Label in name - Screen reader users navigating the page through a mobile device, will hear the button ‘Show 
filters’ announce as ‘Toggle filters’. A visual screen reader user could be confused by the 
different announcement, as it is expected to hear ‘Show filters’. Additionally, ‘Toggle filters’ 
is not descriptive for the purpose of the button as a dialog is displayed rather than content 
toggling. 

- Page 21 - Status message - Changing the state of checkboxes within the filters section of the page results in the text ‘9 
results of 150’ updating to reflect the updating results. The change of results text is not 
audibly read to screen reader users, meaning that the user may be unaware that results 
have updated.
